workshops/Minecraft_Modding/README.md

```diff
@@ -0,0 +1,178 @@
+
+One more thing, make another RegistryObject like last time, but keep it as an `<Item>`. Change the name to something like MY_BLOCK_ITEM. Then make it equal to `ITEMS.register("myblock", () -> new BlockItemBase(MY_BLOCK.get()));`. 
+
+Now within your modname directory, make a new directory called blocks. Now make two classes, one called BlockItemBase, and another with your block name. Within your BlockItemBase just use this snippet as we are just adding it to our mod's tab. 
+
+```java 
+public class BlockItemBase extends BlockItem { 
+    public BlockItemBase(Block block) { 
+        super(block, new Item.Properties().group(kaimod.TAB)); 
+    } 
+} 
+```
+Change kaimod to the name of your mod. 
+
+Now open your block class and this is where we set the properties of the item! At the end of the class name, make sure it extends Block. Now make a method inside of your class with the same name, make it public as well. Within the method write 
+``` java
+super(Block.Properties.create() 
+    ); 
+```
+Within the parenthesis write Material. then your IDE should list all the Materials in the game. This just means what type of block it is. I set mine to `Material.IRON`. Now after the parenthesis put down a period and there should be a lot of different methods available. For the sake of simplicity, I will just make it only minable with a pickaxe. You can do that by setting `.harvestTool(ToolType.Pickaxe)`.  
+
+That is all for coding our block now onto getting it in our game. 
+
+# Part 6: Block Textures 
+
+Making our block have a texture is easy. We already did most of the hard work in the items part. 
+
+The first step is to add onto the `en_US.json` file in `src/main/resources/assets/modname/lang`. Very simply add this line and replaces kaimod, myblock, MyBlock with your names. `"block.kaimod.myblock": "MyBlock"`. Now in your block states folder make a json file with your block as the title. Inside of it add this: 
+``` 
+
+{ 
+  "variants": { 
+    "": { "model": "modname:block/blockname" } 
+  } 
+} 
+```
+Now for the next json, go into `models/block` and add a the same json name as the previous one. Then add this to the json: 
+``` 
+{ 
+  "parent": "block/cube_all", 
+  "textures": { 
+    "all": "modname:blocks/blockname" 
+  } 
+} 
+```
+Now for the final json, go into `models/item` and make another file with the same name and add this: 
+
+``` 
+{ 
+  "parent": "modname:block/blockname" 
+} 
+```
+Remember to replace modname and blockname. Now make a 16x16 png texture, I will be using a simple texture as an example. Place that png under `textures/blocks`. Make sure it has the same name you have been using for the file names. 
+![Simple green/red texture](https://cloud-lhwjz30iu.vercel.app/myblock.png)
+Now you have finished making your first mod. Go start it and have fun. 

```

Comment:
You need to include a final outro section where you show 3 hacks and a proper outro. Ideally you could also recap what you did and learned, show the final project in action, etc.